appveyor.yml

```diff
@@ -1,24 +1,17 @@
-version: "{build}"
 environment:
     CLI_VERSION: latest
     DOTNET_CLI_TELEMETRY_OPTOUT: 1
     DOTNET_SKIP_FIRST_TIME_EXPERIENCE: true
 image: Visual Studio 2019
 configuration:
     - Release
-skip_tags: true
 before_build:
     - dotnet tool install -g GitVersion.Tool
     - dotnet gitversion /l console /output buildserver
 build_script:
     - dotnet build -c Release -p:Version=%GitVersion_NuGetVersion%
 test_script:
     - dotnet test -c Release --no-build
-artifacts:
```

Comment:
I believe this is the change required to publish the symbols. By default Appveyor says that it will publish all `nupkg` artifacts and the related symbols.